src/database/migrations/0008-create-proposal_field_values.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE proposal_field_values (
+  id INTEGER PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+  proposal_version_id INTEGER NOT NULL,
+  application_form_field_id INTEGER NOT NULL,
+  value VARCHAR NOT NULL,
+  sequence INTEGER NOT NULL,
```

Comment:
Four 32-bit integers in a row followed by varchar is probably better packed than three 32-bit integers followed by varchar (64-bit?) followed by 32-bit followed by 64-bit timestamp. In short, sequence and value could be swapped.